.gitignore

```diff
@@ -1,3 +1,4 @@
 *.egg-info
 *.pyc
 pytket/extensions/azure/_metadata.py
+.env
```

Comment:
I would rather not add this to the ignore file: the choice of directory name for a virtual environment is a personal matter, and you can add it to `.git/info/exclude` locally.